src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.Log.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+using static Microsoft.Azure.WebJobs.Script.Grpc.Messages.StreamingMessage;
+
+namespace Microsoft.Azure.WebJobs.Script.Grpc
+{
+    internal partial class GrpcWorkerChannel
+    {
+        // EventId range is 800-899
+        private static class Logger
+        {
+            private static readonly Action<ILogger, string, ContentOneofCase, Exception> _channelReceivedMessage = LoggerMessage.Define<string, ContentOneofCase>(
+                LogLevel.Debug,
+                new EventId(820, nameof(ChannelReceivedMessage)),
+                "[channel] received {workerId}: {msgType}");
+
+            private static readonly Action<ILogger, string, Exception> _invocationResponseReceived = LoggerMessage.Define<string>(
+                LogLevel.Debug,
+                new EventId(821, nameof(InvocationResponseReceived)),
+                "InvocationResponse received for invocation: '{invocationId}'");
+
+            private static readonly Action<ILogger, string, Exception> _ignoringRpcLog = LoggerMessage.Define<string>(
+                LogLevel.Debug,
```

Comment:
Leaving this as `Debug` for now as it may be noisy. My next PR which starts to address the out-of-order messages should reduce this significantly, but at the moment we have no idea how much this is happening.